src/utils/storage.js

```diff
@@ -22,6 +24,21 @@ const storageUtils = {
       OLD_CLOSED_KEY: 'wallet:closed',
       OLD_BACKUP_KEY: 'wallet:backup',
       OLD_TYPE_KEY: 'wallet:type',
+      OLD_NETWORK_KEY: 'wallet:network',
+      OLD_SERVER_KEY: 'wallet:server',
+    }
+
+    if (LOCAL_STORE.getItem(oldStorageKeys.OLD_NETWORK_KEY)) {
+      helpers.updateNetwork(LOCAL_STORE.getItem(oldStorageKeys.OLD_NETWORK_KEY));
+      LOCAL_STORE.removeItem(oldStorageKeys.OLD_NETWORK_KEY);
+    }
+
+    if (LOCAL_STORE.getItem(oldStorageKeys.OLD_SERVER_KEY)) {
+      LOCAL_STORE.setItem(
+        SERVER_KEY,
+        LOCAL_STORE.getItem(oldStorageKeys.OLD_SERVER_KEY),
+      )
+      LOCAL_STORE.removeItem(oldStorageKeys.OLD_SERVER_KEY);
```

Comment:
This will load the network and server from the previous version into the new version and update the redux to show the correct network